inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java

```diff
@@ -192,6 +192,7 @@ private List<DataConfig> fetchNonFileTasks(TaskRequest taskRequest) {
             sourceEntity.setAgentIp(taskRequest.getAgentIp());
             sourceEntity.setUuid(taskRequest.getUuid());
             if (sourceMapper.updateByPrimaryKeySelective(sourceEntity) == 1) {
+                sourceEntity.setVersion(sourceEntity.getVersion() + 1);
```

Comment:
Suggest changing to `++sourceEntity.getVersion()`.

Reply:
Cannot proceed` ++sourceEntity.getVersion(`

Reply:
OK👌